scripts/version.sh

```diff
@@ -0,0 +1,11 @@
+# this scripts runs after npm version is ran
+VERSION_FILE="./lib/version.ts";
+DIST_FILE="./dist/search-insights.min.js";
+
+PACKAGE_VERSION=$(cat ./package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
```

Comment:
Could we dot it in JavaScript? It would be a bit simpler no?

Reply:
I'm later writing in a file, calling git and calling yarn, and those part are simpler (or more readable) in shell to me 😄

Reply:
With JavaScript you can call bash command easily like the ones below.